src/dashboard/src/media/js/ingest.js

```diff
@@ -242,20 +233,24 @@ $(function()
             this.$('.job-detail-actions').append($select);
           }
 
-          if ('Approve normalization' == this.model.get('type'))
+          // Approve normalization
+          if ('de909a42-c5b5-46e1-9985-c031b50e9d30' == this.model.get('link_id'))
```

Comment:
I believe that we can make this snippet clearer by adding some verbosity, e.g.

`approve_normalization_mscl = 'de909a42-c5b5-46e1-9985-c031b50e9d30'`

Or a comment highlighting the `de909a4` is the UUID for approve normalization from the MS chainlink table. 

Second, is there a reason for the construct to be:

`if 'constant' == 'value returned from function'` - I would normally expect it to be written:

`if (this.model.get('link_id') == 'value_we_want_to_confirm') `

(This seems to be the same for most of the calls below so this comment applies to those as well. My concern is that UUIDs are in a lot of places in Archivematica. If we can be as explicit as possible about the source of a UUID to future readers or folks new to the code (even if it is obvious to us), then it saves a little time for them.)